Wikipedia/Code/ArticleLocationAuthorizationCollectionViewCell.swift

```diff
@@ -40,9 +40,9 @@ class ArticleLocationAuthorizationCollectionViewCell: ArticleLocationExploreColl
         var origin = CGPoint(x: layoutMargins.left, y: size.height + spacing - layoutMargins.bottom)
         let widthForLabels = size.width - layoutMargins.left - layoutMargins.right
         let authorizeSpacing = 3 * spacing
-        origin.y += authorizeTitleLabel.wmf_preferredHeight(at: origin, maximumWidth: widthForLabels, horizontalAlignment: .center, spacing: authorizeSpacing, apply: apply)
-        origin.y += authorizeButton.wmf_preferredHeight(at: origin, maximumWidth: widthForLabels, minimumWidth: min(300, widthForLabels), horizontalAlignment: .center, spacing: authorizeSpacing, apply: apply)
-        origin.y += authorizeDescriptionLabel.wmf_preferredHeight(at: origin, maximumWidth: widthForLabels, horizontalAlignment: .center, spacing: authorizeSpacing, apply: apply)
+        origin.y += authorizeTitleLabel.wmf_preferredHeight(at: origin, maximumWidth: widthForLabels, horizontalAlignment: .left, spacing: authorizeSpacing, apply: apply)
```

Comment:
Is this correct on RTL?

Reply:
It looks like you can move the `isDeviceRTL` up into `CollectionViewCell` and use that here to define a `labelHorizontalAlignment` local variable to set for these two labels